cmd/nonce-service/main.go

```diff
@@ -89,28 +81,16 @@ func main() {
 		c.NonceService.DebugAddr = *debugAddr
 	}
 
-	// TODO(#6610): Remove once we've moved to derivable prefixes by default.
-	if c.NonceService.NoncePrefix != "" && c.NonceService.UseDerivablePrefix {
-		cmd.Fail("Cannot set both 'noncePrefix' and 'useDerivablePrefix'")
-	}
-
-	// TODO(#6610): Remove once we've moved to derivable prefixes by default.
-	if c.NonceService.UseDerivablePrefix && c.NonceService.NoncePrefixKey.PasswordFile == "" {
-		cmd.Fail("Cannot set 'noncePrefixKey' without 'useDerivablePrefix'")
-	}
-
-	if c.NonceService.UseDerivablePrefix && c.NonceService.NoncePrefixKey.PasswordFile != "" {
-		key, err := c.NonceService.NoncePrefixKey.Pass()
-		cmd.FailOnError(err, "Failed to load 'noncePrefixKey' file.")
-		c.NonceService.NoncePrefix, err = derivePrefix(key, c.NonceService.GRPC.Address)
-		cmd.FailOnError(err, "Failed to derive nonce prefix")
-	}
+	key, err := c.NonceService.NoncePrefixKey.Pass()
```

Comment:
If `c.NonceService.NoncePrefixKey.PasswordFile` is absent, the `Pass()` method _does not_ throw an error because 
```
// Make PasswordConfigs optional, for backwards compatibility.
	if pc.PasswordFile == "" {
		return "", nil
	}
```

What happens is that the nonce services come up, but integration tests fail with
```
acme.messages.Error: urn:ietf:params:acme:error:badNonce :: The client sent an unacceptable anti-replay nonce :: JWS has an invalid anti-replay nonce: "whateverxxxxx"
```

Reply:
That's sneaky! I've re-added the check for this config key's presence.